ui/libexec/dockerutils.sh

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
+
+
+# Helpers for spammy sections
+img_build_fail() {
+    error "Failed to build the installer image. Check the log for details."
+    die "You may need to run '$0 --clean' to clean up Docker"
+}
+
+img_pull_fail() {
+    die "Failed to pull the installer image. Check the log for details."
+    die "You may need to run '$0 --clean' to clean up Docker"
+}
+
+ui_tarball() {
```

Comment:
Before creating the UI tarball we need to make sure the `ui/inventory.py` file has execute permissions. Otherwise the "step1" will fail with the following error.

```
[centos@adrian-ecsce-sn1 ~]$ step1
[sudo] password for centos:
ecs-install> Initializing data container, one moment ...
ecs-install> OK
ecs-install> Applying deploy.yml
> Do you accept the license? ["yes" or "no"]: yes
Great, thanks for helping us keep the lawyers happy!
ERROR! The file /usr/local/src/ui/inventory.py looks like it should be an executable inventory script, but is not marked executable. Perhaps you want to correct this with `chmod +x /usr/local/src/ui/inventory.py`?
Operation failed.
ERROR! The file /usr/local/src/ui/inventory.py looks like it should be an executable inventory script, but is not marked executable. Perhaps you want to correct this with `chmod +x /usr/local/src/ui/inventory.py`?
Operation failed.
```

I confirm that adding the execute permissions to the file.

```
chmod +x ui/inventory.py
``` 

and re running the bootstrap script and step1 fixes the issue.

@padthaitofuhot where would be the right place to add the permissions?